homework.md

```diff
@@ -4,110 +4,109 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
```

Comment:
Dear Hattice, replace the multiply operator * back into the original question otherwise 2 _ 3 _ 6 _ 1 _ 10 \* 34 does not make any sense. Remove _ and \
Thank you

Reply:
Thanks for your comments Delroy. I haven't changed the question. Prettier is changing asterixis to underscore automatically. I am going to have a look at it.

Reply:
@Hatice06 did you manage to fix this issue with VS Code auto-formatting? Let me know if you need any help

Reply:
I searched it on the internet and found that the Prettier is causing the problem. I disabled it and saved the file again. However it didn't work. If you can help me, I would be grateful.